notebooks/IV-queries.sparqlbook

```diff
@@ -0,0 +1,176 @@
+    "metadata": {}
+  },
+  {
+    "kind": 1,
+    "language": "markdown",
+    "value": "##### IVGR7-LMB-2: De gemeenteraad neemt kennis van de ontvankelijkheid van de gezamenlijke akte van voordracht en verklaart de voorgedragen kandidaat-schepenen verkozen in de rangorde van de gezamenlijke akte van voordracht van kandidaat-schepenen",
+    "metadata": {}
+  },
+  {
+    "kind": 2,
+    "language": "sparql",
+    "value": "PREFIX org: <http://www.w3.org/ns/org#>\nPREFIX ext: <http://mu.semte.ch/vocabularies/ext/>\nPREFIX lmb: <http://lblod.data.gift/vocabularies/lmb/>\nPREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>\nPREFIX persoon: <http://data.vlaanderen.be/ns/persoon#>\nPREFIX foaf: <http://xmlns.com/foaf/0.1/>\nPREFIX skos: <http://www.w3.org/2004/02/skos/core#>\nSELECT DISTINCT ?rang ?naam ?status ?begindatum_mandaat ?einddatum_mandaat ?opvolger WHERE {\n  # TODO: replace this by the correct 'bestuursperiode'\n  ?bestuursorgaan lmb:heeftBestuursperiode <http://data.lblod.info/id/concept/Bestuursperiode/a2b977a3-ce68-4e42-80a6-4397f66fc5ca>.\n  ?bestuursorgaan org:hasPost ?mandaat.\n  # Filter on 'schepen'\n  ?mandaat org:role <http://data.vlaanderen.be/id/concept/BestuursfunctieCode/5ab0e9b8a3b2ca7c5e000014>.\n  ?mandataris org:holds ?mandaat.\n  ?mandataris mandaat:isBestuurlijkeAliasVan ?persoon.\n  # TODO: ?rangorde is a string or number?\n  ?mandataris mandaat:rangorde ?rang.\n  ?mandataris mandaat:status/skos:prefLabel ?status.\n  ?persoon persoon:gebruikteVoornaam ?voornaam.\n  ?persoon foaf:familyName ?achternaam.\n  BIND(CONCAT(?voornaam, \" \", ?achternaam) AS ?naam)\n  # TODO: This should be the date of the meeting\n  BIND(NOW() as ?begindatum_mandaat)\n  VALUES ?einddatum_mandaat { undef }\n  # TODO: unsure how we will fetch the 'opvolger'\n  VALUES ?opvolger { undef }\n\n} \n# TODO: Unsure how we will sort 'rang'\nORDER BY ?rang\n# TODO: remove this limit in production\nLIMIT 10",
+    "metadata": {}
+  },
+  {
+    "kind": 1,
+    "language": "markdown",
+    "value": "##### IVGR7-LMB-3: De gemeenteraad neemt kennis van de ontvankelijkheid van de gezamenlijke akte van voordracht en verklaart de voorgedragen kandidaat-schepenen verkozen in de rangorde van de gezamenlijke akte van voordracht van kandidaat-schepenen",
```

Comment:
I can't say I understand what this one is about at all :s

Reply:
I can maybe add some english translations (not sure how helpful they will be). But as the source document is also in dutch, it might be hard to interpret the queries.